src/hooks/useAllLiquidtyMiningCampaigns.ts

```diff
@@ -13,15 +13,17 @@ import { getAddress, parseUnits } from 'ethers/lib/utils'
 import { useKpiTokens } from './useKpiTokens'
 import { PairsFilterType } from '../components/Pool/ListFilter'
 
+// Fragments are not possible, they will be resovled
+const CAMPAIGN_REWARDS_TOKEN_COMMON_FIEDLDS = ['address: id', 'name', 'symbol', 'decimals', 'derivedNativeCurrency']
+
+const CAMPAIGN_COMMON_FIEDLDS = ['duration', 'startsAt', 'endsAt', 'locked', 'stakingCap', 'stakedAmount']
+
```

Comment:
Also wondering why you choose to do these common fields like this? Can this be done more cleanly with fragment?https://www.apollographql.com/docs/react/data/fragments/